### ISSUE TITLE
[Lint] Small warnings

### ISSUE THREAD

**Issue by ntjandra** [ntjandra/RPG-bot#2]
Linter caught these things to fix for cleaner code...
```
./abilities.py:2:1: F403 'from data.import_data import *' used; unable to detect undefined names
./abilities.py:8:1: E302 expected 2 blank lines, found 1
./abilities.py:9:5: F405 'characters' may be undefined, or defined from star imports: data.import_data
./abilities.py:9:40: W291 trailing whitespace
./abilities.py:10:10: E222 multiple spaces after operator
./abilities.py:14:1: E302 expected 2 blank lines, found 1
./abilities.py:17:10: E211 whitespace before '('
./abilities.py:21:1: E302 expected 2 blank lines, found 1
./abilities.py:24:59: E261 at least two spaces before inline comment
./abilities.py:25:10: E211 whitespace before '('
./abilities.py:26:10: E211 whitespace before '('
./abilities.py:30:1: E302 expected 2 blank lines, found 1
./abilities.py:46:15: W291 trailing whitespace
./abilities.py:52:13: F405 'skills' may be undefined, or defined from star imports: data.import_data
./abilities.py:53:14: F405 'characters' may be undefined, or defined from star imports: data.import_data
./abilities.py:58:53: W291 trailing whitespace
./abilities.py:59:1: W293 blank line contains whitespace
./bot.py:2:1: F401 'asyncio' imported but unused
./bot.py:4:1: F401 'requests' imported but unused
./bot.py:5:1: F401 'shutil' imported but unused
./bot.py:9:1: F403 'from data.import_data import *' used; unable to detect undefined names
./bot.py:18:4: E111 indentation is not a multiple of 4
./bot.py:20:1: E302 expected 2 blank lines, found 1
./bot.py:26:1: E266 too many leading '#' for block comment
./bot.py:27:1: E302 expected 2 blank lines, found 1
./bot.py:62:1: E266 too many leading '#' for block comment
./bot.py:63:1: E302 expected 2 blank lines, found 1
./bot.py:68:23: E211 whitespace before '('
./bot.py:74:37: E251 unexpected spaces around keyword / parameter equals
./bot.py:74:81: E251 unexpected spaces around keyword / parameter equals
./bot.py:74:83: E251 unexpected spaces around keyword / parameter equals
./bot.py:76:27: F405 'characters' may be undefined, or defined from star imports: data.import_data
./bot.py:83:1: E266 too many leading '#' for block comment
./bot.py:84:1: E302 expected 2 blank lines, found 1
./bot.py:89:23: E211 whitespace before '('
./bot.py:107:1: E266 too many leading '#' for block comment
./bot.py:108:1: E302 expected 2 blank lines, found 1
./bot.py:111:23: E211 whitespace before '('
./bot.py:117:28: F405 'save_player' may be undefined, or defined from star imports: data.import_data
./bot.py:[12](https://github.com/ntjandra/RPG-bot/actions/runs/5932732304/job/16087055744?pr=1#step:5:13)0:11: W291 trailing whitespace
./bot.py:122:1: E266 too many leading '#' for block comment
./bot.py:123:1: E302 expected 2 blank lines, found 1
./bot.py:126:23: E211 whitespace before '('
./bot.py:[13](https://github.com/ntjandra/RPG-bot/actions/runs/5932732304/job/16087055744?pr=1#step:5:14)2:28: F405 'load_player' may be undefined, or defined from star imports: data.import_data
./bot.py:135:11: W291 trailing whitespace
./bot.py:137:1: E266 too many leading '#' for block comment
./bot.py:138:1: E302 expected 2 blank lines, found 1
./bot.py:[14](https://github.com/ntjandra/RPG-bot/actions/runs/5932732304/job/16087055744?pr=1#step:5:15)1:19: F405 'init_party' may be undefined, or defined from star imports: data.import_data
./data/import_data.py:5:1: E302 expected 2 blank lines, found 1
./data/import_data.py:12:1: E305 expected 2 blank lines after class or function definition, found 1
./data/import_data.py:16:1: E302 expected 2 blank lines, found 1
./data/import_data.py:17:17: F541 f-string is missing placeholders
./data/import_data.py:19:14: E211 whitespace before '('
./data/import_data.py:20:46: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:20:48: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:20:51: W291 trailing whitespace
./data/import_data.py:22:11: F541 f-string is missing placeholders
./data/import_data.py:26:1: E302 expected 2 blank lines, found 1
./data/import_data.py:29:14: E211 whitespace before '('
./data/import_data.py:30:57: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:30:59: E251 unexpected spaces around keyword / parameter equals
./data/import_data.py:30:62: W291 trailing whitespace
./data/import_data.py:36:1: E302 expected 2 blank lines, found 1
./data/import_data.py:41:1: W293 blank line contains whitespace
./data/import_data.py:46:[15](https://github.com/ntjandra/RPG-bot/actions/runs/5932732304/job/16087055744?pr=1#step:5:16): 
```

**Comment by HARXINE** [comment]
Do you like to use docstring or not? I will also docstring all your comments

**Comment by HARXINE** [comment]
I'm done and I'll just say that you know more about docstring and not more than 12 @ntjandra

**Comment by ntjandra** [comment]
Hi @HARXINE, thanks a bunch. I'd love your help with resolving these. Originally, this project is just something I made for a friend group when we played an RPG-like game (DnD but with less rolling in combat). Neat to see you forked it and I'd love to invite you to contribute. 

This is still a just a quick and dirty solution, and I agree with you that docstrings are better than '#'  for comments. Some of the ### is a result of me copy pasting some markdown notes I had and then coding after. 

I'll take a look your PR #3 soon!
